changelog/README.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 21th October 2022
+* Extended [Get rate pricing](../operations/rates.md#get-rate-pricing) response with `AmountPrices` and `AmountPrices` in [Resource category pricing](../operations/rates.md#resource-category-pricing).
```

Comment:
I would suggest this for the Changelog:

* Deprecated `BasePrices` and replaced it with `BaseAmountPrices` in the response to [Get rate pricing](../operations/rates.md#get-rate-pricing); `BaseAmountPrices` includes information about the tax breakdown
* Deprecated `Prices` and replaced it with `AmountPrices` in the response to [Get rate pricing](../operations/rates.md#get-rate-pricing); `AmountPrices` includes information about the tax breakdown

Reply:
Not sure about the second one since `Prices` are nested inside of `Resource category pricing`.

Reply:
I don't think it matters that `Prices` is nested inside `Resource category pricing` because the name is unique on that page. Alternatively, you could say "Deprecated `Prices` (in `Resource category pricing`) and replaced it...". The third bullet point in the Changelog can now be deleted altogether.

Reply:
Ok, I will leave it as it is since looks like it's more similar how we write it.